docs/chapters.html

```diff
@@ -49,7 +49,15 @@
           3: 'A dangerous quest',
           8: 'The attack'
         },
-      })
+      });
+
+      const playerAutoVTT = cloudinary.videoPlayer('player-auto-vtt', {
```

Comment:
```suggestion
      const playerByNameVTT = cloudinary.videoPlayer('player-by-name-vtt', {
```
And where relevant down in the code snippet itself.